airflow/config_templates/default_airflow.cfg

```diff
@@ -538,6 +538,9 @@ statsd_host = localhost
 statsd_port = 8125
 statsd_prefix = airflow
 
+# optional comma-separated prefix whitelist (e.g: scheduler,executor,dagrun)
```

Comment:
Worth adding the docs from `metrics.rst` here too:

```
If you want to avoid send all the available metrics to StatsD, you can configure a whitelist of prefixes to send only
the metrics that start with the elements of the list
```